atef/widgets/happi.py

```diff
@@ -51,6 +51,7 @@ class HappiSearchWidget(DesignerDisplay, QWidget):
     _client: Optional[happi.client.Client]
     _last_selected: List[str]
     _search_thread: Optional[ThreadWorker]
+    _tree_view_expanded: Optional[List[QtCore.QModelIndex]]
```

Comment:
Since we're stashing `QModelIndex` with , it probably should be cleared when the happi model updates underneath it, I'd imagine. Probably as simple as setting `_tree_view_expanded` in `refresh_happi`